packages/react/src/components/TwoColumnPanel/TwoColumnPanel.tsx

```diff
@@ -0,0 +1,244 @@
+import React, {
+  forwardRef,
+  useState,
+  cloneElement,
+  isValidElement,
+  useRef,
+  useLayoutEffect,
+  useEffect
+} from 'react';
+import { useId } from 'react-id-generator';
+import FocusTrap from 'focus-trap-react';
+import Icon from '../Icon';
+import Tooltip from '../Tooltip';
+import ClickOutsideListener from '../ClickOutsideListener';
+import { ColumnLeft, ColumnRight } from './';
+import classnames from 'classnames';
+
+interface TwoColumnPanelProps extends React.HTMLAttributes<HTMLDivElement> {
+  initialCollapsed?: boolean;
+  showCollapsedPanelLabel?: string;
+  hideCollapsedPanelLabel?: string;
+}
+
+const TwoColumnPanel = forwardRef<HTMLDivElement, TwoColumnPanelProps>(
+  (
+    {
+      className,
+      children,
+      initialCollapsed = false,
+      showCollapsedPanelLabel = 'Show Panel',
+      hideCollapsedPanelLabel = 'Hide Panel',
```

Comment:
I don't think we should provide default values for this, since they wouldn't be "internationalized". I think we should instead make them required so the consumer knows they are responsible for providing a translated string.

Reply:
There's a couple of other places where we provide default text values:

* https://github.com/dequelabs/cauldron/blob/3127ddebe7d616863d47dcbe76cd4f30589db5fa/packages/react/src/components/Dialog/index.tsx#L39
* https://github.com/dequelabs/cauldron/blob/3127ddebe7d616863d47dcbe76cd4f30589db5fa/packages/react/src/components/Pagination/index.tsx#L33-L37
* https://github.com/dequelabs/cauldron/blob/3127ddebe7d616863d47dcbe76cd4f30589db5fa/packages/react/src/components/Pointout/index.tsx#L56-L57
* https://github.com/dequelabs/cauldron/blob/3127ddebe7d616863d47dcbe76cd4f30589db5fa/packages/react/src/components/SkipLink/index.tsx#L23-L24

This provides at a minimum accessible by default since the lack of a label would be an accessibility violation. I hope anyone using it provides a translated string but it would be hard to enforce that from cauldron, so at a minimum ensuring that some label is sets a baseline of accessibility.

Reply:
> Interesting. Wouldn't making them required have the same effect of making sure we have a label?

Not necessarily.

If they don't use typescript, we can't enforce it with typescript. PropTypes isn't really an enforcement as it just prompts a console log in devtools when rendering about missing props.